pkg/reconciler/contour/resources/httpproxy.go

```diff
@@ -300,8 +302,20 @@ func MakeHTTPProxies(ctx context.Context, ing *v1alpha1.Ingress, serviceToProtoc
 				}
 
 				hostProxy.Name = kmeta.ChildName(ing.Name+"-"+class+"-", host)
-				hostProxy.Spec.VirtualHost = &v1.VirtualHost{
-					Fqdn: host,
+
+				if cfg.Contour.CORSPolicy != nil {
+					corsPolicy := &v1.CORSPolicy{}
+					tmp, _ := json.Marshal(cfg.Contour.CORSPolicy)
+					json.Unmarshal(tmp, &corsPolicy)
```

Comment:
we shouldn't do this marshalling - just use the `v1.CORSPolicy` once in the config